doc/release-notes.rst

```diff
@@ -2,17 +2,364 @@
  Release Notes
 ===============
 
-v12.1.0 Luminous (RC)
+v12.1.1 Luminous (RC)
 =====================
 
-This is the first release candidate for Luminous, the next long term
-stable release.
+This is the second release candidate for Luminous, the next long term stable
+release.
 
 Ceph Luminous (v12.2.0) will be the foundation for the next long-term
 stable release series.  There have been major changes since Kraken
 (v11.2.z) and Jewel (v10.2.z), and the upgrade process is non-trivial.
 Please read these release notes carefully.
 
+Notable Changes since v12.1.1 (RC1)
+-----------------------------------
```

Comment:
I'd rather put this section down below and keep all the top-level luminous release info at the top, so that this is treated more like a draft for the final luminous release notes.  That way people still see (at the top) the major changes, upgarde procedure, etc.